### PR TITLE
Python logstash

### PR DESCRIPTION
**Pull request by sy-hbk**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/python-logstash/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  version: "{{ version }}"
+
+ source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 10943e5df83f592b4d61b63ad1afff855ccc8c9467f78718f0a59809ba1fe68c
+
+ build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (xhochy) [inline recipes/python-logstash/meta.yaml]: No need to include the arguments, they're here as default now.

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```

File: recipes/python-logstash/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ about:
+  home: https://github.com/vklochan/python-logstash
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Python logging handler for Logstash.
+
+ extra:
+  recipe-maintainers:
+    - pg-hbk
```
Comment (xhochy) [inline recipes/python-logstash/meta.yaml]: @pg-hbk are you ok with being a maintainer?

Comment (pg-hbk) [reply recipes/python-logstash/meta.yaml]: Yes

File: recipes/python-logstash/LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (isuruf) [inline recipes/python-logstash/LICENSE]: This file is unnecessary as conda-build will find it from the source repo.

**Pull request merged**